### DISCUSSION TITLE
what is tauri2 support minimum android version? and is it support android 10?

### DISCUSSION THREAD

**Discussion by tu6ge** [tauri-apps/tauri#11843] [Q&A]
I want to create an app in android 10, but I ran into a problem, the screen on Android 10 was always white, and the console reported the following error
![WX20241202-142305@2x](https://github.com/user-attachments/assets/7ebb32a8-49c7-4022-8c83-ba5f87493487)

**Comment by tu6ge** [comment]
I am developing a new app ,and run `pnpm tauri android dev` ,and link remote device developing, the error is remote device

**Comment by FabianLars** [comment]
In theory 7 and above, but i only saw 9 and above being tested by someone i think. That said, the emulators for older versions also have older webview versions than typical real world devices so they often don't work well to test.

**Reply by tu6ge** [reply]
My android 10 's webview is chorme 77, and it is not support vite runtime. so ,it is not support remote developing and testing.
